src/sentry/api/helpers/group_index.py

```diff
@@ -929,18 +929,35 @@ def update_groups(request, group_ids, projects, organization_id, search_fn):
 
     if "assignedTo" in result:
         assigned_actor = result["assignedTo"]
+        assigned_by = request.data.get("assignedBy")
```

Comment:
Should we validate `assignedBy` somehow?

Reply:
@wedamija yea I added a check if it is within a list of accepted values. `None` is acceptable bc it can be reassigned by users through the API or through one of the other endpoints that use `update_groups` function